Backends/RasterImage/output-to-image.lisp

```diff
@@ -44,6 +44,10 @@
 
 (defmacro with-output-to-rgba-image ((stream-var image &rest options)
                                      &body body)
+  `(with-output-to-iamge (,stream-var ,image ,@options) ,@body))
```

Comment:
`with-output-to-image` (typo)

Reply:
either way macro `with-output-to-rgba-image` should be removed. symbol is not exported neither it is used anymore.

Reply:
Done.